en/02_Developer_Guides/01_Templates/04_Rendering_Templates.md

```diff
@@ -194,5 +203,62 @@ class MyPageController extends PageController
 }
 ```
 
-> [!WARNING]
-> A common mistake is trying to loop over an array directly in a template - this won't work. You'll need to wrap the array in some `ModelData` instance as mentioned above.
+Or wrap the data in a `ModelData` subclass:
```

Comment:
```suggestion
Or wrap the data in a `ModelData` subclass such as `ArrayList`:
```

Reply:
Done, but said `ArrayData` since that's what the example shows.